pkg/templates/notificationtemplates/notification_generator_test.go

```diff
@@ -0,0 +1,97 @@
+package notificationtemplates_test
+
+import (
+	"testing"
+
+	"github.com/codeready-toolchain/host-operator/pkg/templates/assets"
+	"github.com/codeready-toolchain/host-operator/pkg/templates/notificationtemplates"
+
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+func TestGetNotificationTemplates(t *testing.T) {
+	logf.SetLogger(zap.Logger(true))
+	t.Run("ok", func(t *testing.T) {
+		t.Run("get notification templates", func(t *testing.T) {
+			// when
+			template, found, err := notificationtemplates.GetNotificationTemplates("userprovisioned", notificationtemplates.WithEmptyCache())
```

Comment:
Does the first call to this function require additional parameters beyond the template name?  If this is so, can it be initialized during controller startup?

Reply:
No, just the name.The options are to clear cache (if needed) or used for testing..

Reply:
removed opts here: 2afd35318551e963d97e30d90eec4ac4e08c3b26